nimble/host/src/ble_gattc.c

```diff
@@ -4413,6 +4413,120 @@ ble_gatts_notify(uint16_t conn_handle, uint16_t chr_val_handle)
     return rc;
 }
 
+int
+ble_gatts_notify_multiple_custom(uint16_t conn_handle,
+                                 uint16_t chr_count,
+                                 struct ble_gatt_notif *tuples)
+{
+#if !MYNEWT_VAL(BLE_GATT_NOTIFY_MULTIPLE)
+    return BLE_HS_ENOTSUP;
+#endif
+
+    int rc;
+    int i = 0;
+    uint16_t cur_chr_cnt = 0;
+    /* mtu = MTU - 1 octet (OP code) */
+    uint16_t mtu = ble_att_mtu(conn_handle) - 1;
+    struct os_mbuf *txom = ble_hs_mbuf_att_pkt();
+    struct ble_hs_conn *conn;
+
+    conn = ble_hs_conn_find(conn_handle);
+    if (conn == NULL) {
+        return ENOTCONN;
+    }
+
+    if ((conn->bhc_gatt_svr.peer_cl_sup_feat[0] & 0x04) == 0) {
+        BLE_HS_LOG_DEBUG("not sup\n");
```

Comment:
this contradicts with description:
" * If GATT client doesn't support receiving multiple handle notifications,
 * this will use GATT notification for each characteristic, separately."